internal/cephfs/driver.go

```diff
@@ -164,5 +164,12 @@ func (fs *Driver) Run(conf *util.Config) {
 		util.WarningLogMsg("EnableGRPCMetrics is deprecated")
 		go util.StartMetricsServer(conf)
 	}
+	if conf.EnableProfiling {
+		if !conf.EnableGRPCMetrics {
+			go util.StartMetricsServer(conf)
+		}
+		util.WarningLogMsg("Registering profiling handler")
```

Comment:
it should be the info log not warning log?

Reply:
yes. will update this.